src/modules/launcher/Wox.Infrastructure/Helper.cs

```diff
@@ -148,6 +148,16 @@ public static Process OpenInConsole(string path)
             return Process.Start(processStartInfo);
         }
 
+        public static bool OpenCommandInShell(string path, string pattern, string arguments, string workingDir = null, ShellRunAsType runAs = ShellRunAsType.None, bool runWithHiddenWindow = false)
+        {
+            if (pattern.Contains("%1", StringComparison.Ordinal))
+            {
+                arguments = pattern.Replace("%1", arguments);
+            }
```

Comment:
What would happen if the pattern doesn't contain %1? Would it work as well?

Reply:
If there are no %1 we will use `arguments` as is which is the best we can do, I suppose.

Reply:
This shouldn't happen as %1 is the default place holder for parameters on file associations. If %1 is missing you can't open the file by double click or context menu in explorer.

Reply:
Got it. THank you!